src/main/kotlin/io/zerobase/smarttracing/GraphDao.kt

```diff
@@ -53,4 +57,183 @@ class GraphDao(private val driver: Driver) {
             }
         }
     }
+
+    /**
+     * Creates an organization node inside neo4j.
+     *
+     * @param organization name of the organization that is being created.
+     * @param phone phone number for the contact for the organization.
+     * @param email email for the contact for the organization.
+     * @param contactName name of the contact for the organization.
+     * @param address address of the organization.
+     * @param testing if the organization owns sites that can do testing.
+     * @param multiSite reporting if the organization has multiple sites.
+     *
+     * @return organization id.
+     */
+    fun createOrganization(organization: String, phone: String, email: String,
+                           contactName: String, address: String, testing: Boolean,
+                           multiSite: Boolean): OrganizationId? {
+        val validNum = phoneUtil.isValidNumber(phoneUtil.parseAndKeepRawInput(phone, "US"))
+
+        if (!validNum) {
+            return null
+        }
```

Comment:
while i approve of validating the phone numbers, we need to at least log something about the phone number not being valid. just returning `null` doesn't allow the UI to communicate to the user why the request failed. let's throw a custom exception and have an exception mapper convert it to the http response we want